client/components/data/query-jetpack-modules/test/query-jetpack-modules.jsx

```diff
@@ -58,8 +63,9 @@ describe( 'fetchModuleList', () => {
 		expect( mockFetchModuleList ).not.toHaveBeenCalled();
 	} );
 
-	test( "Ensure we're calling fetchModuleList only for atomic sites", async () => {
+	test( "Ensure we're calling fetchModuleList for Atomic sites", async () => {
 		isWpcomAtomic = true;
+		isJetpack = false;
```

Comment:
Doesn't seem like a valid scenario. AFAIK there can't be a working Atomic site that isn't a Jetpack site under the hood.